l1-contracts/contracts/bridgehub/ChainAssetHandler.sol

```diff
@@ -146,6 +146,10 @@ contract ChainAssetHandler is
                 bridgehubBurnData.chainId,
                 bridgehubBurnData.ctmData
             );
+
+            if (_settlementChainId != L1_CHAIN_ID && BRIDGEHUB.chainTypeManager(_settlementChainId) != ctm) {
+                revert SLHasDifferentCTM();
```

Comment:
Let's use the same error message [there](https://github.com/matter-labs/era-contracts/pull/1605/files#diff-7aed1eba45994a84e007c4d938981b77395648e8dbfd4e0b965dab2255bb524bR315)?